sdk/attestation/attestation/test/utils/Buffer.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Buffer as buffer } from "safe-buffer";
```

Comment:
Curious, why did you choose `safe-buffer` over the built-in `buffer` given we support node.js 8 as a minimum version?

It doesn't look like we're importing buffer from this file in all of our other test files so elsewhere we're using the regular Buffer.

Reply:
I copied the code for decoding JWT from the jws and this is how they do it. I did not want to take chances whether there is a difference between safe-buffer and the built-in one, especially that I am short on time.